### PR TITLE
ops: align versions in monorepo

### PR DESCRIPTION
**Pull request by Kavinjsir**
https://github.com/rilldata/rill/issues/5592

### CODE REVIEW

**Comment by himadrisingh** [comment]
lets make them `0.51.0` as the next release would be cut from this branch.

**Review by ericpgreen2 — COMMENTED** [review]
Given 1) we're not publishing any of our packages to npm, 2) we have our own way of managing code versions via git tags, and 3) we're not consuming any `version` anywhere.... what if we removed the `version` fields altogether?

Rather:
- We could add `"private": true` to our package.json files to be more explicit that they're not published to `npm`.
- We could cite `web-common` in our `devDependencies` like: `"web-common": "workspace:*"`, which seems to be a standard practice in monorepos.

Thoughts?

**Comment by Kavinjsir** [comment]
> Given 1) we're not publishing any of our packages to npm, 2) we have our own way of managing code versions via git tags, and 3) we're not consuming any `version` anywhere.... what if we removed the `version` fields altogether?
> 
> Rather:
> 
>     * We could add `"private": true` to our package.json files to be more explicit that they're not published to `npm`.
> 
>     * We could cite `web-common` in our `devDependencies` like: `"web-common": "workspace:*"`, which seems to be a standard practice in monorepos.
> 
> 
> Thoughts?

@ericpgreen2 That sounds great to me! It helps reduce the complexity of version maintenance across different workspaces and during the release process. I also tried update `version: "0.0.0"` for each workspace in the project, in an attempt to explicitly "lock" the version value in package.json.
One issue here is, when specifying `"web-common": "workspace:*"`, it would crash by the following error:
workspace-issue

Not quite an expert over here, so I tried replace `"workspace:*"` with `"*"` to resolve the issue.
Here is the reference I got: https://stackoverflow.com/a/79135939

How would you like this?

cc: @himadrisingh I'm following @ericpgreen2 's suggestion which looks a good approach to reduce complexity. I also updated the version in the entry package.json to `0.51.0` regarding your suggestion :)

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@rilldata/dev",
-  "version": "0.10.0",
+  "version": "0.51.0",
```
Comment (ericpgreen2) [inline package.json:3]: My comments on not relying on any `version` also apply to this root package.json. What if we set this one to `"0.0.0"` too?

File: web-auth/package.json [hunk 2]
```diff
@@ -1,6 +1,7 @@
 {
 	"name": "web-auth",
-	"version": "0.0.1",
+  "private": true,
```
Comment (ericpgreen2) [inline web-auth/package.json:3]: Redundant `private: true` (see line 5)

File: package.json (hunk 1 above)
Comment (Kavinjsir) [reply package.json]: Sounds good!

File: web-auth/package.json (hunk 2 above)
Comment (Kavinjsir) [reply web-auth/package.json]: Good Catch!

**Review by ericpgreen2 — APPROVED** [review]
LGTM! This seems like a good set-it-and-forget it approach for now. If we ever change how we version our releases, or if we ever publish some packages to npm, then we can revisit.

**Pull request merged**